bin/wp_import_db.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+# Title:		F1 WP Import DB
+# Description:	This script will connect to the WordPress DEV|STAGE|LIVE environment, export an .sql file 
+#				copy of it's database and import that one into the project's Vagrant 
+#				environment. A time saver tool for development.
+# Author:		Forum One
+# Author URI:	https://forumone.com
+# Version:		1.1.0
+# Usage:		- 1) Adjust values within the REMOTE CONFIG section below.
+#				- 2) From the Vagrant project <root>, run ./bin/wp_import_db.sh
+
+
+# ======================================== REMOTE CONFIG ======================================== #
+# Adjust below variable values as needed.
+# {DEV|STAGE|LIVE} Details:
+USER_AT_HOST="user@webhost.com"							# User and host where to connect via SSH.
+WEB_ROOT="/var/www/vhosts/project_name.dev/public/"		# WordPress remote environment web root.
+# ======================================== END REMOTE CONFIG ======================================== #
+
+
+# ======================================== COMMON CONFIG ======================================== #
+SCRIPT_BASE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+VM_WEB_ROOT=$SCRIPT_BASE_DIR"/../public/"
+
+# Vagrant VM Details:
+VAGRANT_USER_AT_HOST="vagrant@127.0.0.1"
+VAGRANT_PORT=2222
+VAGRANT_SSH_ID_FILE=$SCRIPT_BASE_DIR"/../.vagrant/machines/default/virtualbox/private_key"
+VAGRANT_VM_WEB_ROOT="/vagrant/public/"
+# ======================================== END COMMON CONFIG ======================================== #
+
+
+
+# ======================================== FUNCTIONS ======================================== #
+# Processes the database migration from the remote environment to the local Vagrant VM.
+function f1_run {
+
+	# REMOTE ENV WORKFLOW:
+	# 1- Exports the DB on remote environment.
+	echo ">>> Connecting to $USER_AT_HOST:$WEB_ROOT environment and exporting the database..." &&
+	SSH_OUTPUT_RESULT=$( ssh $USER_AT_HOST "cd $WEB_ROOT &&
```

Comment:
Don't dump the db to the public document root - perhaps the parent directory a better location?